### ISSUE TITLE
Contents of label.txt file for handwriting recognition model

### ISSUE THREAD

**Issue by Lilyane1** [tulasiram58827/ocr_tflite#9]
Hi, my name is lilian from Kenya . I am an university student currently working on an application that does handwriting and text recognition . I trained and converted this code https://keras.io/examples/vision/handwriting_recognition/ into a tflite file and from my research i need the model.tflite file and a label.txt file to use the trained model in my application, But i do not know what should be in the label.txt file of my model. Kindly help me
